CHANGELOG.md

```diff
@@ -8,6 +8,7 @@ Next
 * Added Travis-CI - [@dblock](https://github.com/dblock).
 * Added TODO list and enhanced README documentation - [@ptitvert](https://github.com/ptitvert)
 * [#15](https://github.com/dblock/oshi/pull/15), [#18](https://github.com/dblock/oshi/pull/18): Added support for CPU load - [@kamenitxan](https://github.com/kamenitxan), [@Sorceror](https://github.com/Sorceror).
+* Include inactive memory in mac/linux available - [@dbwiddis](https://github.com/dbwiddis).
```

Comment:
Unfortunately I didn't have the pull request # until I committed (I suppose I could have guessed).   And since I've been committing via svn, I am really unfamiliar with the git command lines to do this. Argh.  I'll do my best, I guess... you're really making me have to learn new tools!